<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Capture2</class>
 <widget class="QMainWindow" name="Capture2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="openFButton">
        <property name="text">
         <string>&amp;open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveFButton">
        <property name="text">
         <string>&amp;save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TimeL">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>&amp;exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>choose NIC</string>
        </property>
        <property name="buddy">
         <cstring>NIChooser</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="NIChooser">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CaptureButton">
        <property name="text">
         <string>begin</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DetailButton">
        <property name="text">
         <string>show &amp;detail</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="FilterButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&amp;Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="MainTable">
        <property name="accessibleName">
         <string/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>230</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <item>
         <property name="text">
          <string>hu</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>menubar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>911</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>866</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CaptureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>NIChooser</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CaptureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>FilterButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>1050</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
